crates/js-component-bindgen/src/intrinsics.rs

```diff
@@ -114,6 +119,117 @@ pub fn render_intrinsics(
 
     for i in intrinsics.iter() {
         match i {
+            Intrinsic::AsyncifyAsyncInstantiate => output.push_str("
+                const asyncifyModules = [];
+                let asyncifyPromise;
+                let asyncifyResolved;
+                async function asyncifyInstantiate(module, imports) {
+                  const instance = await instantiateCore(module, imports);
+                  const memory = instance.exports.memory || (imports && imports.env && imports.env.memory);
+                  const realloc = instance.exports.cabi_realloc || instance.exports.cabi_export_realloc;
+                  if (instance.exports.asyncify_get_state && memory) {
```

Comment:
What if we don't have a memory here or asyncify_get_state? Does the fallthrough work, or should we throw an error?